source/sass/buyers-guide/views/homepage.scss

```diff
@@ -1,5 +1,6 @@
 #view-home {
   $breakpoint: $bp-md;
+  $cutoff-offset: 88px;
```

Comment:
Can we add a comment that explains why this is 88, and not 87 or 89? (or some other value =D)

Reply:
Sure I can add that in but it won't be super useful since pixel values all came from design 😄

Reply:
comment added